### ISSUE TITLE
After training, model has bad detections or lower accuracy.

### ISSUE THREAD

**Issue by zzundark** [Stick-To/Object-Detection-API-Tensorflow#2]
I am trained models(centernet, refinedet and yolov3) under argumentation configuration...

image_augmentor_config = {
    'data_format': 'channels_last',
    'output_shape': [320, 320],
    # 'zoom_size': [330, 330],
    # 'crop_method': 'random',
    'flip_prob': [0.0, 0.5],
    'fill_mode': 'BILINEAR',
    'keep_aspect_ratios': False,
    'constant_values': 0.,
    'color_jitter_prob': 0.5,
    # 'rotate_range': [0.5, -10., 10.],
    'pad_truth_to': 60,
}

dataset: VOC 2007

batch size: 16 

buffer size :1024

config: code's default except batch size(because gpu's vram size..)

The model's training is working well....but after training, model has bad detections or lower accuracy.

additionally, I don't know why model needs param "pad_truth_to"..

Did I somthing wrong?

**Comment by Stick-To** [comment]
the config in file does not provide the best configuration for training, just show how to fill in configuration files, may try different hyperparameters or try to pretrain the model could get the better result.
The results could be sess here:
https://github.com/Stick-To/YOLO-tensorflow
https://github.com/Stick-To/CenterNet-tensorflow
https://github.com/Stick-To/RefineDet-tensorflow


And the param "pad_truth_to", different images have different number of objects, in my implemention, different images just share same featrue extractor,  but the losses are calculated image by image and then  
average them

**Comment by Stick-To** [comment]
I am sorry, there is a small bug in Object-Detection-API-Tensorflow Repository, it is fixed now

**Comment by Stick-To** [comment]
here is the Repository  for https://github.com/Stick-To/CenterNet-tensorflow alone
